charts/rstudio-pm/README.md

```diff
@@ -116,7 +116,7 @@ The Helm `config` values are converted into the `rstudio-pm.gcfg` service config
 | awsAccessKeyId | bool | `false` | awsAccessKeyId is the access key id for s3 access, used also to gate file creation |
 | awsSecretAccessKey | string | `nil` | awsSecretAccessKey is the secret access key, needs to be filled if access_key_id is |
 | command | bool | `false` | command is the pod's run command. By default, it uses the container's default |
-| config | object | `{"HTTP":{"Listen":":4242"},"Metrics":{"Enabled":true},"Server":{"RVersion":"/opt/R/3.6.2/"}}` | config is a nested map of maps that generates the rstudio-pm.gcfg file |
+| config | object | `{"HTTP":{"Listen":":4242"},"Metrics":{"Enabled":true},"Server":{"RVersion":"/opt/R/4.2.0/bin/R"}}` | config is a nested map of maps that generates the rstudio-pm.gcfg file |
```

Comment:
We also ran across this issue and set manually as done here; thanks for adding to the upstream chart!

Note: per the docs [here](https://docs.posit.co/rspm/admin/building-packages/?h=bin+r#r-configuration), the `bin/R` is not recommended
```suggestion
| config | object | `{"HTTP":{"Listen":":4242"},"Metrics":{"Enabled":true},"Server":{"RVersion":"/opt/R/4.2.0/"}}` | config is a nested map of maps that generates the rstudio-pm.gcfg file |
```

Reply:
Use the default version (symlink) inside the container.
```suggestion
| config | object | `{"HTTP":{"Listen":":4242"},"Metrics":{"Enabled":true},"Server":{"RVersion":"/opt/R/default/"}}` | config is a nested map of maps that generates the rstudio-pm.gcfg file |
```